### ISSUE TITLE
A11y shrine: purpose of each link

### ISSUE THREAD

**Issue by felixzapata** [PolymerElements/app-layout#366]
### Description

The purpose of each link should be clear from the link text
### Expected outcome

Every button or link text should have a clear title, name or description to describe its purpose.
### Actual outcome

There are buttons and links over the application without any text inside it. Some of them grouped elements within them or show only an icon.
